open_instruct/finetune.py

```diff
@@ -237,7 +237,7 @@ def main(args: FlatArguments):
             splits=["train"],
             save_data_dir=args.dataset_mix_dir,
             columns_to_keep=["messages"],
-            need_columns=["messages"],
+            # need_columns=["messages"],
```

Comment:
Why do this? Should we just remove the whole arg (including from `get_datasets`)?

Reply:
I'm probably going to -- I'm just testing this for now and I had to push the change to rebuild the docker image. tbd